lib/iris/tests/unit/coords/test_Coord.py

```diff
@@ -324,6 +324,12 @@ def test_numeric_nd(self):
         with self.assertRaises(ValueError):
             coord.collapsed()
 
+    def test_collapsed_overflow(self):
+        coord = DimCoord(points=np.array([1493892000, 1493895600, 1493899200],
```

Comment:
these numbers came from a real scenario, hence why I used them.  the key characteristic is that each value is valid, but the combination of the two of them overflows
that is the 'bug' that I wanted to fix and this test explicitly examines that characteristic

so, on reflection, I'd rather keep them